content/en/blog/2024/in-cluster-operator-deprecation-announcement/index.md

```diff
@@ -32,6 +32,12 @@ Users who prefer the operator pattern for running Istio can migrate to either of
 
 Helm migration requires translating your `IstioOperator` YAML into a Helm `values.yaml` file.  Tooling to support this migration will be provided alongside the Istio 1.24 release.
 
+Translate an IstioOperator YAML file into Helm values and a shell script to install equivalent Helm charts:
+
+{{< text bash >}}
+$ istioctl manifest translate -f istio.yaml
+{{< /text >}}
+
```

Comment:
```suggestion
Helm migration requires translating your `IstioOperator` YAML into Helm values. Istio 1.24 and above includes a `manifest translate` command to perform this operation. The output is a `values.yaml` file, and a shell script to install equivalent Helm charts.

{{< text bash >}}
$ istioctl manifest translate -f istio.yaml
{{< /text >}}

```

Reply:
(side note: this should also end up in the actual docs somewhere, which I don't think it ever did)